fail2ban/server/filter.py

```diff
@@ -898,22 +898,22 @@ def textToIp(text, useDns):
 		return ipList
 
 	@staticmethod
-	def cidr(i, n):
-		""" Convert an IP address string with a CIDR mask into a 32-bit
-			integer.
+	def netAddr2Dec(ipstring, cidr):
+		""" Convert a string IPv4 address with CIDR
+		to decimal IP address of first address in block (network address).
 		"""
 		# 32-bit IPv4 address mask
 		MASK = 0xFFFFFFFFL
-		return ~(MASK >> n) & MASK & DNSUtils.addr2bin(i)
+		return ~(MASK >> cidr) & MASK & DNSUtils.addr2dec(ipstring)
 
 	@staticmethod
-	def addr2bin(string):
-		""" Convert a string IPv4 address into an unsigned integer.
+	def addr2dec(ipstring):
```

Comment:
>    My thinking was, do you get back a binary string or an integer? Can that
>    more accurately be described as "bin" or "dec"?

>    Binary value of IP Address "10.0.0.0" is "00001010 00000000 00000000
>    00000000". Hex: 0A:00:00:00 

both above are specific "views" of such a binary number

> Decimal: 167772160.

and this one a typical decimal printout of an integer as well.  If it
was a string with a "view" in integer form -- it would have been 'dec',
but now it is just an integer, which is just stored as a number and we
use digital computers which internally use binary arithmetics and
storage...  that is what was meant behind "bin" here -- just pure binary
representation, not really a "view/printout" which as if it was a
string -- it can be hex or any other view if someone "prints it out".
So I would just vote to leave it alone as is -- 'dec' would be imho more
confusing that helpful.